package.json

```diff
@@ -21,8 +24,10 @@
   "author": "Arsalan Khattak <akkhattak65@gmail.com>",
   "license": "MIT",
   "dependencies": {
+    "@types/inquirer": "^7.3.0",
```

Comment:
Make sure to put typings in devDependancies, as they are never needed to compile!